packages/manager/apps/hpc-vmware-managed-vcd/src/pages/listing/datacentres/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { DataGridTextCell, Links } from '@ovhcloud/manager-components';
```

Comment:
Following folder structure, the file shouldn't be named ListingDatacentres.page.tsx ?

Reply:
will be managed in #12477